### PR TITLE
 Rename and redirect urls to be seasonX/episodeX/ 

### LINKED ISSUE #97: Change URLs to seasonX/episodeX/ format & add /seasonX/ landing pages.
From Raphael on Slack: "I would suggest folders for the seasons. So it would be /season1/episode1/ and /season2/episode1/. Thinking ahead, then it will be easier in years to build a crawlable archive, if the site ever will that big its needed. And we could link in the navigation to season pages, if we want to create them, at some point not all Episodes will be able to live on the homepage for page load reasons."

### PR DESCRIPTION
**Pull request by stephaniehobson**
- Give all episodes new URLs and aliases for the old ones
- Rename files to reflect new season/episode structure
- Rename asset folders to reflect new season/episode structure
- Add season summary pages
- Tried and failed to do away with `number`, it is passed as part of the tracking to iTunes and RadioPublic and since the bonus episodes do not have a `number` auto generating it would change the number assigned to anything after S01E07

I was pleasantly surprised at how easy it was to create url aliases... and then I went overboard. Let me know if you'd prefer these commits were submitted separately.

The diffs look intimidating because a lot of white space got stripped by my editor. I can try to undo that if necessary.

Closes #97

### CODE REVIEW

**Comment by stephaniehobson** [comment]
I belatedly realized there isn't actually an issue open for this. I'll create that on Monday.

File: layouts/partials/episode-module.html [hunk 1]
```diff
@@ -1,22 +1,22 @@
-{{ $.Scratch.Set "imagePath" (printf "episodes/%s/artwork.png" (string .Params.number)) }}
-{{ $.Scratch.Set "imagePathHighRes" (replace (.Scratch.Get "imagePath") ".png" "-high-res.png") }}
-{{ $.Scratch.Set "itunesTracking" (printf "episode%s" (string .Params.number)) }}
-{{ $.Scratch.Set "rpTracking" (printf "episode%s-modal" (string .Params.number)) }}
-{{ if in .Params.number "B" }}
+{{ if in .Params.seasonepisode "B" }}
```
Comment (jpetto) [inline layouts/partials/episode-module.html]: Maybe add comments here and in the `else if` below to make it obvious what conditions are being checked, e.g. bonus episodes and trailers.

**Comment by jpetto** [comment]
Oh, and you can add `?w=1` to the end of any list of PR files to ignore whitespace changes. 😄

**Comment by jpetto** [comment]
We should also check dev to ensure`alias` is performing the correct type of redirect (`301`, I believe).

**Comment by stephaniehobson** [comment]
It's a `301` on dev which is the correct one.

**Pull request merged**